src/assets/menu-scroll.js

```diff
@@ -0,0 +1,33 @@
+window.onscroll = function() {
```

Comment:
There are a few ways to do this, and this in one of the worst... :)

For starters, `window.onscroll = headerShowHide;` would be preferable -- there's no need for the extra function overhead.  However, this will still clobber any other `onscroll` behaviours that might be added from elsewhere in the codebase (or, indeed, anywhere else).  The best option is (pretty much always) `window.addEventListener("scroll", headerShowHide);`.